modules/silicon/scripts/build/container/jupyterlab/cloudbuild.yaml

```diff
@@ -0,0 +1,32 @@
+timeout: 3600s
+substitutions:
+  _OPENLANE_VERSION: 2022.02.01_02.19.58
+steps:
+- name: 'gcr.io/cloud-builders/git'
+  args: ['clone', '-b', $_OPENLANE_VERSION, 'https://github.com/The-OpenROAD-Project/OpenLane']
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: '/bin/bash'
+  args:
+  - '-c'
+  - |-
+    apt-get update && apt install -yq python3-venv
+    python3 -m venv env
+    env/bin/python -m pip install pyyaml click
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: '/bin/bash'
+  env:
+  - EXTERNAL_PDK_INSTALLATION=0
+  - NO_PDKS=0
+  args:
+  - '-c'
+  - |-
+    source env/bin/activate
+    docker pull gcr.io/$PROJECT_ID/openlane-pdk:$_OPENLANE_VERSION || make -C OpenLane OPENLANE_IMAGE_NAME=gcr.io/$PROJECT_ID/openlane-pdk:$_OPENLANE_VERSION openlane
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/openlane-jupyterlab:$BUILD_ID', '--build-arg', 'PROJECT_ID=$PROJECT_ID', '--build-arg', 'OPENLANE_VERSION=$_OPENLANE_VERSION', '.']
```

Comment:
that should point to the `Dockerfile` with `-f` since this is launched from the module root.

Reply:
Done